source/blog/2018-07-10-Skydive-With-oVirt.html.md

```diff
@@ -5,22 +5,21 @@ tags: oVirt, oVirt 4.2, open source virtualization,network
 date: 2018-07-10 09:01:00 UTC
 ---
 
-## Benefit to oVirt users
-oVirt users may now analyze and see thier network topology via the Skydive UI. 
-Skydive allows the users to see the result of their configuration done via oVirt.
-They may capture traffic from VM1 to VM2 for example or monitor the traffic between two VMs or hosts. 
-Also they may generate some traffic between 2 running VMs on different hosts for example.
-Users may create alerts in Skydive UI to notify that some traffic disconnected or down.
-
 [Skydive network](http://skydive.network/) is an open source real-time network topology and protocols analyzer providing a comprehensive way of understanding what is happening in your network infrastructure.
 The common use cases will be, troubleshooting, monitoring, SDN integration and much more.
 It has features such as:
 * Topology capturing - Captures network topology, interface, bridge and more
 * Flow capture - Distributed probe, L2-L4 classifier, GRE, VXLAN, GENEVE, MPLS/GRE, MPLS/UDP tunnelling support
 * Extendable - Support for external SDN Controllers or container based infrastructure, OpenStack. Supports extensions through API
 
+## Benefit to oVirt users
+Skydive allows oVirt administrators to see the network configuration and topology of their oVirt cluster.
+Administrators can capture traffic from VM1 to VM2 or monitor the traffic between VMs or hosts. 
+Skydive can generate traffic between 2 running VMs on different hosts and then analyze.
+Administrators can create alerts in Skydive UI to notify when traffic is disconnected or down.
+
 ## Installation steps
-1. git clone https://github.com/skydive-project/skydive.git
+1. 'git clone https://github.com/skydive-project/skydive.git'
```

Comment:
almost :) you used single quote, but it needs backtick. backtick is the angled one.
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting
(on my standard qwerty keyboard, backtick is under the Esc key)